guides/common/modules/con_generating-reports-by-using-the-project-mcp-server.adoc

```diff
@@ -0,0 +1,10 @@
+:_mod-docs-content-type: CONCEPT
+
+[id="generating-reports-by-using-the-project-mcp-server"]
+= Generating reports by using the {Project} MCP server
```

Comment:
Afaik we won't really brand the mcp server itself, should we hardcode `{Project}` here? This would apply to multiple places here